web/ASC.Web.Client/src/Shell.jsx

```diff
@@ -417,7 +446,9 @@ const Shell = ({ items = [], page = "home", ...rest }) => {
               <PublicRoute exact path={WIZARD_URL} component={WizardRoute} />
               <PrivateRoute path={ABOUT_URL} component={AboutRoute} />
               {loginRoutes}
+              <Route path={ENTER_CODE_URL} component={EnterCodeRoute} />
```

Comment:
Route must be hidden if roomsMode == false

Reply:
Resolve: https://github.com/ONLYOFFICE/AppServer/pull/517/commits/da121a1bfeef2311768464a0929cfbdac7b6ad98